pkg/Microsoft.DotNet.ILCompiler/targets/Microsoft.NETCore.Native.Publish.targets

```diff
@@ -0,0 +1,66 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
This seems to be a copy of the files from src\BuildIntegration . Is it intentional?

Reply:
Yes. The versions in this folder have properties wrapped in targets with an explicit dependency on the "SetupProperties" target in the `Microsoft.DotNet.ILCompiler.targets` - this is to ensure that the name and location of the runtime package are resolved before the properties are defined and any build/publish steps which use their values are run. The reason for the copy is to avoid unnecessary bloating of the original BuildIntegration files, which will be used once the workaround can be removed. 
Do you think modifying the original files is a better approach?

Reply:
> Do you think modifying the original files is a better approach?

I think so. Just add comments around the workarounds so that we have a good way to find it.